fiftyone/core/dataset.py

```
@@ -2450,6 +2642,42 @@ def _rename_sample_fields(self, field_mapping, view=None):
             sample_collection, paths, new_paths
         )
 
+        label_schemas = self.label_schemas
+        new_label_schemas = copy.deepcopy(label_schemas)
+        active_label_schemas = self.active_label_schemas
+        for path, new_path in zip(paths, new_paths):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)